packages/salesforce-adapter/src/filters/workflow.ts

```diff
@@ -218,6 +235,24 @@ const filterCreator: FilterCreator = () => {
 
       originalWorkflowChanges = await groupByAsync(allWorkflowRelatedChanges, getWorkflowApiName)
 
+      if (client === undefined) {
+        // In the SFDX flow we need to get the entire flow to avoid part of it being dropped.
```

Comment:
Nit: this took me an extra second to orient myself
```suggestion
        // In the SFDX flow we need to get the entire workflow to avoid part of it being dropped.
```